src/iso15118/d20/state/session_stop.cpp

```diff
@@ -45,11 +45,14 @@ Result SessionStop::feed(Event ev) {
 
         if (req->ev_termination_code.has_value()) {
             logf_info("EV termination code: %s", req->ev_termination_code.value().c_str());
+            m_ctx.ev_termination.ev_termination_code = req->ev_termination_code.value().c_str();
```

Comment:
Why no local `ev_termination` variable?

Reply:
`Ev_Information` is used in various code places. `ev_termination` is only used once. So a local variable makes here more sense